kornia/feature/laf.py

```diff
@@ -0,0 +1,381 @@
+# flake8: noqa E127
+# flake8: noqa E128
+from typing import Tuple
+
+import kornia
+import torch
+import math
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import warnings
+try:
+    import matplotlib.pyplot as plt
+    MATPLOTLIB = True
+except:
+    warnings.warn("no matplotlib found, visualize_LAF not available ")
+    MATPLOTLIB = False
+    
+
+def angle_to_rotation_matrix(angle: torch.Tensor,
+                             do_deg2rad: bool = False) -> torch.Tensor:
+    """
+    Creates a rotation matrix out of angles
+    Args:
+        angle: (torch.Tensor): tensor of angles, any shape.
+        do_deg2rad: (bool): if we should convert to radians first
+
+    Returns:
+        torch.Tensor: tensor of Nx2x2 rotation matrices.
+
+    Shape:
+        - Input: :math:`(N)`
+        - Output: :math:`(N, 2, 2)`
+
+    Example:
+        >>> input = torch.rand(1, 3)  # Nx3
```

Comment:
I do not understand the meaning of the N here

Reply:
You are commenting on an outdated version. Now it is * instead of N there